frontend/src/views/WheresMyGene/components/GeneSearchBar/index.tsx

```diff
@@ -61,7 +63,13 @@ export default function GeneSearchBar({
       });
       setTissues(newTissues);
     }
-  }, [rawTissues, filteredTissues, selectedOrganismId, setTissues]);
+  }, [
+    rawTissues,
+    filteredTissues,
+    selectedOrganismId,
+    setTissues,
+    selectedTissues,
```

Comment:
@atarashansky did you install https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint ? The plugin would catch that for you!